src/nfa/operations.cc

```diff
@@ -573,6 +597,7 @@ bool Mata::Nfa::is_in_lang(const Nfa& aut, const Run& run)
 
 /// Checks whether the prefix of a string is in the language of an automaton
 // TODO: slow and it should share code with is_in_lang
+//method?
```

Comment:
If a method, it should be a method of `Run`. Otherwise, we could make it a method of Nfa by calling it something like `Nfa::Nfa::accepts_prefix(Run run)` or `Nfa::Nfa::has_prefix_in_lang(Run run)`. Calling the function otherwise reads as NFA is prefix in language of run which does not make any sense to me.

Reply:
as above